src/tabs/Tabs.stories.tsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as React from "react";
+import {storiesOf} from "@storybook/react";
+import {Tab, TabOrientation, TabType} from ".";
+
+const tabsData = [
+    {
+        tabName: "Dashboard",
+        disableTab: false,
```

Comment:
do we really need tab word in every prop ?  name, disabled, component -> converys same meaning.  @akate1  - please advise on this.

Reply:
Done.